datahub-web-react/src/app/search/EntityGroupSearchResults.tsx

```diff
@@ -0,0 +1,90 @@
+import { ArrowRightOutlined } from '@ant-design/icons';
+import { Button, Card, Divider, List, Space, Typography } from 'antd';
+import * as React from 'react';
+import { useHistory } from 'react-router-dom';
+import { SearchCfg } from '../../conf';
+import { useGetSearchResultsQuery } from '../../graphql/search.generated';
+import { EntityType } from '../../types.generated';
+import { IconStyleType } from '../entity/Entity';
+import { useEntityRegistry } from '../useEntityRegistry';
+import { navigateToSearchUrl } from './utils/navigateToSearchUrl';
+
+const styles = {
+    header: { marginBottom: 20 },
+    resultHeaderCardBody: { padding: '16px 24px' },
+    resultHeaderCard: { right: '52px', top: '-40px', position: 'absolute' },
+    resultList: { width: '100%', borderColor: '#f0f0f0', marginTop: '12px', padding: '16px 32px' },
+    seeAllButton: { fontSize: 18 },
+    resultsContainer: { width: '100%', padding: '40px 132px' },
+};
+
+interface Props {
+    type: EntityType;
+    query: string;
+}
+
+export const EntityGroupSearchResults = ({ type, query }: Props) => {
+    const history = useHistory();
+    const entityRegistry = useEntityRegistry();
+    const { data } = useGetSearchResultsQuery({
+        variables: {
+            input: {
+                type,
+                query,
+                start: 0,
+                count: SearchCfg.RESULTS_PER_PAGE,
+                filters: null,
```

Comment:
should we be propagating filters here as well?

Reply:
Ah, I see- we can't support facet filtering on all results 👍

Reply:
correcto!!